superset-frontend/src/filters/components/Select/SelectFilterPlugin.tsx

```diff
@@ -163,6 +171,14 @@ export default function PluginFilterSelect(props: PluginFilterSelectProps) {
             </Option>
           );
         })}
+        {currentSuggestionSearch &&
+          !(values || []).some(
```

Comment:
WDYT about using `ensureIsArray(values)` instead of `values || []`? I'm not sure if it'd be easier to read, but maybe a bit safer in case `values` is not an array

Reply:
Good idea 👍